### DISCUSSION TITLE
What do you use to reset Tap layer to Base layer?

### DISCUSSION THREAD

**Discussion by gogamid** [manna-harbour/miryoku#335] [Q&A] [answered]
I use corne with Miryoku ZMK, i cannot find documentation on how to come back from Tap mode to Base, basically reset.
Is there already built in combo that i am not aware of? If not, then what is your suggestion?

**Comment by manna-harbour** [comment, marked as answer]
See https://github.com/manna-harbour/miryoku/tree/master/docs/reference#additional-features:

> Returning from Tap requires a reset or adding a custom keybinding.

All keys on Tap should behave as regular single action keys without any delay. So it's not possible to add tap-hold, tap dance, combos etc. by default as they will all add delays to the tap. If that is not an issue in your case you can add one of those, including substituting a custom layer with https://github.com/manna-harbour/miryoku/discussions/85 if required. If you have unused keys on your keyboard (e.g. a 6-column Corne) you can instead substitute a custom mapping as above.

For Miryoku ZMK, the binding that needs to be added is `&u_to_U_BASE`. In general, it can be found by looking at e.g. https://github.com/manna-harbour/miryoku_zmk/blob/e6683e9f8b6c199b339208b1b501e88a7308ed48/miryoku/miryoku_babel/miryoku_layer_alternatives.h#L276C58.

**Reply by gogamid** [reply]
Than you a lot
